tests/test_deepedit_transforms.py

```diff
@@ -14,81 +14,303 @@
 import numpy as np
 from parameterized import parameterized
 
-from monai.apps.deepedit.transforms import ClickRatioAddRandomGuidanced, DiscardAddGuidanced, ResizeGuidanceCustomd
+from monai.apps.deepedit.transforms import (
+    AddGuidanceFromPointsCustomd,
+    AddGuidanceSignalCustomd,
+    AddInitialSeedPointMissingLabelsd,
+    AddRandomGuidanceCustomd,
+    DiscardAddGuidanced,
+    FindAllValidSlicesMissingLabelsd,
+    FindDiscrepancyRegionsCustomd,
+    NormalizeLabelsInDatasetd,
+    ResizeGuidanceMultipleLabelCustomd,
+    SingleLabelSelectiond,
+    SplitPredsLabeld,
+    ToCheckTransformd,
+)
+from monai.utils import min_version, optional_import
 from monai.utils.enums import PostFix
 
-IMAGE = np.array([[[[1, 0, 2, 0, 1], [0, 1, 2, 1, 0], [2, 2, 3, 2, 2], [0, 1, 2, 1, 0], [1, 0, 2, 0, 1]]]])
-LABEL = np.array([[[[0, 0, 0, 0, 0], [0, 1, 0, 1, 0], [0, 0, 1, 0, 0], [0, 1, 0, 1, 0], [0, 0, 0, 0, 0]]]])
+measure, _ = optional_import("skimage.measure", "0.14.2", min_version)
```

Comment:
@Nic-Ma

Reply:
Hi @diazandr3s ,

Please add your test here to skip `min` tests as it has optional import:
https://github.com/Project-MONAI/MONAI/blob/dev/tests/min_tests.py#L168

Thanks.

Reply:
Many thanks, @Nic-Ma